soccer/gameplay/timeout_behavior.py

```diff
@@ -0,0 +1,89 @@
+import composite_behavior
+import behavior
+import time
+import enum
+
+## This class adds a timeout to any subbehavior
+#
+# The way this works is by wrapping your desired behavior in this one.
+# To use, simply add the behavior you want to timeout like this:
+#
+# self.add_subbihavior(timeout_behavior.TimeoutBehavior. timeout),
+# instead of normal timeouts
```

Comment:
oh yah thats a bit confusing, I'll change it to `timeout_behavior.TimeoutBehavior(instance_of_your_behavior)`